content/en/docs/hertz/tutorials/basic-feature/middleware/csrf.md

```diff
@@ -0,0 +1,341 @@
+---
+title: "CSRF"
+date: 2022-11-30
+weight: 2
+description: >
+
+---
+
+Cross-site request forgery（CSRF）is an attack method to coerce users to perform unintended operations on currently logged in Web applications.
+
+
+Hertz provides [CSRF](https://github.com/hertz-contrib/csrf) middleware that helps you prevent cross-site request forgery attacks.
+
+## Install
+
+``` shell
+go get github.com/hertz-contrib/csrf
+```
+
+## Example
+
+```go
+package main
+
+import (
+	"context"
+
+	"github.com/cloudwego/hertz/pkg/app"
+	"github.com/cloudwego/hertz/pkg/app/server"
+	"github.com/hertz-contrib/csrf"
+	"github.com/hertz-contrib/sessions"
+	"github.com/hertz-contrib/sessions/cookie"
+)
+
+func main() {
+	h := server.Default()
+
+	store := cookie.NewStore([]byte("secret"))
+	h.Use(sessions.Sessions("csrf-session", store))
+	h.Use(csrf.New())
+
+	h.GET("/protected", func(c context.Context, ctx *app.RequestContext) {
+		ctx.String(200, csrf.GetToken(ctx))
+	})
+
+	h.POST("/protected", func(c context.Context, ctx *app.RequestContext) {
+		ctx.String(200, "CSRF token is valid")
+	})
+
+	h.Spin()
+}
+
+```
+
+## Config
+
```

Comment:
I think it would be nice to add a table to list all the configuration fields and give a short introduction, you can refer to the documentation of other middleware.